cluster-manifests/region2/bootstrapping/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
w00t!  Looks good.  (You can resolve this right away)

Reply:
<3